fixture/commercial-license-cert.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I just copied this certificate file from https://scalardb.scalar-labs.com/docs/latest/scalar-licensing/?scalardl-ledger=scalardl-ledger#scalardl-ledger .